src/index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class=
+    "page__calendar calendar calendar--start-day--wen calendar--month-length-30">
```

Comment:
we separate modifier with 2 --, there should be calendar--month-length--30 modifier instead of yours